types/bbob__plugin-helper/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for @bbob/plugin-helper 3.0
+// Project: https://github.com/JiLiZART/bbob
+// Definitions by: shmee <https://github.com/shme-e>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export * from "./char";
+export * from "./helpers";
+export * from "./TagNode";
```

Comment:
https://github.com/JiLiZART/BBob/blob/6ca06b9204d80f4cd59bd49ff9960ad9ee3bb4c7/packages/bbob-plugin-helper/src/index.js#L3C1-L3C37
Please review the export, original one seems to export named symbol only

Reply:
looking into it theres no need to export the other function in TagNode so I just removed that - I'd rather export the types and interfaces required for TagNode